Src/MoneyFox.Windows/CortanaFunctions.cs

```diff
@@ -0,0 +1,226 @@
+﻿using System;
+using System.Threading.Tasks;
+using Windows.Storage;
+using Windows.Media.Capture;
+using Windows.ApplicationModel.VoiceCommands;
+using MoneyFox.Foundation.Resources;
+using System.Xml.Linq;
+
+
+namespace MoneyFox.Windows
+{
+    public static class CortanaFunctions
+    {
+        static MediaCapture mc;
+        static ApplicationDataContainer LocalSettings = ApplicationData.Current.LocalSettings;
+        static async Task EnableMicrophone()
+        {
+            try
+            {
+
+                 mc = new MediaCapture();
+                await mc.InitializeAsync();
+              ///  await win.Launcher.LaunchUriAsync(new Uri("ms-settings:privacy-microphone"));
+                          
+            }
+            catch (Exception e)
+            {
+                StorageFolder storageFolder = ApplicationData.Current.LocalCacheFolder;
+                StorageFile file = await storageFolder.CreateFileAsync("errors.txt", CreationCollisionOption.OpenIfExists);
+                await FileIO.AppendTextAsync(file, e.ToString());
+                await FileIO.AppendTextAsync(file, Environment.NewLine);
+            }
+        }
+        public static async Task IntializeCortana()
+        {
+
+
+            try
+            {
+                await EnableMicrophone();
+
+            }
+            catch (Exception e)
+            {
+
+                StorageFolder storageFolder = ApplicationData.Current.LocalCacheFolder;
+                StorageFile file = await storageFolder.CreateFileAsync("errors.txt", CreationCollisionOption.OpenIfExists);
+                await FileIO.AppendTextAsync(file, e.ToString());
+                await FileIO.AppendTextAsync(file, Environment.NewLine);
+
+            try
+            {
+
+                await CreateVCDfileAndInstallAsync();
+
+            }
+            catch (Exception e)
+            {
+                StorageFolder storageFolder = ApplicationData.Current.LocalCacheFolder;
```

Comment:
As far as I can see, you cannot create here another storageFolder, since you already have one above and this is the same scope.